From 2a0feddbd39f97e86a04448abd0ca6fc52be70cf Mon Sep 17 00:00:00 2001
From: Andrew Cooper <andrew.cooper3@citrix.com>
Date: Wed, 8 Nov 2023 14:47:10 +0000
Subject: [PATCH 007/106] docs/sphinx: Fix syntax issues in exclude-list.rst

sphinx-build reports:

  docs/misra/exclude-list.rst:50: WARNING: Inline emphasis start-string without end-string.

'*' either needs escaping, or put in a literal block.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
Reviewed-by: Luca Fancellu <luca.fancellu@arm.com>
Acked-by: Stefano Stabellini <sstabellini@kernel.org>
Release-acked-by: Henry Wang <Henry.Wang@arm.com>
(cherry picked from commit ab03b284b4f2fbf405fcd821105c85e1a38d314d)
---
 docs/misra/exclude-list.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/misra/exclude-list.rst b/docs/misra/exclude-list.rst
index c97431a861..0266c8b2fe 100644
--- a/docs/misra/exclude-list.rst
+++ b/docs/misra/exclude-list.rst
@@ -42,5 +42,5 @@ Here is an explanation of the fields inside an object of the "content" array:
 
 To ease the review and the modifications of the entries, they shall be listed in
 alphabetical order referring to the rel_path field.
-Excluded folder paths shall end with '/*' in order to match everything on that
+Excluded folder paths shall end with ``/*`` in order to match everything on that
 folder.
-- 
2.25.1

